packages/utils/crypto/src/key.ts

```diff
@@ -20,6 +22,8 @@ export abstract class Keypair {
 	toPeerId(): Promise<PeerId> {
 		throw new Error("Not implemented");
 	}
+
+	// TODO: Should we add not implemented errors for .create and and .from as well?
 }
```

Comment:
@marcus-pousette What do you think - errors for `Key.create()` and `Key.from()` ?

Reply:
might be tricky to do with the types. Because it has to be generic. For example if there is a from(PeerID) that works for publickeys, that type make no sense for bytekey for example. 

I would say dont add until its necessary. (Lazy approach usually is the way)